Sources/AppcuesKit/AppcuesKit.docc/AppcuesFrameViewConfiguring.md

```diff
@@ -0,0 +1,37 @@
+# Configuring AppcuesFrameViews
+
+Using `AppcuesFrameView` instances in your layouts will allow you to embed inline Appcues experience content in your app. This style of pattern is non-modal in the user's experience, differing from modals and tooltips used in mobile flows. Any number of desired embedded experiences can be rendered in the application at any given time.
+
+## Adding AppcuesFrameView in UIKit
+
+When using UIKit for your application layouts, insert ``AppcuesFrameView`` instances wherever you would like Appcues embedded experience content to potentially appear. By default, these views will not take up any space in the rendered layout. Only when qualified experience content is targeted to these frames will they actually be visible. You can think of this process as reserving placeholder locations in your application UI for potential additional content. An ``AppcuesFrameView`` can be used in either Storyboards or coded UI.
+
+## Registering an AppcuesFrameView with the Appcues SDK
```

Comment:
I'd make this a tertiary heading (###), since it's a detail of the secondary "Adding AppcuesFrameView in UIKit"